plugin/external/@wordpress/elements/index.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
@youknowriad @aduth I think this comment was lost, but I think we should use singular `element` here.